src/rest/DiscordAPIError.js

```diff
@@ -34,6 +34,15 @@ class DiscordAPIError extends Error {
      * @type {number}
      */
     this.httpStatus = status;
+
+    /**
+     * The data associated with the request that caused this error
+     * @type {HTTPErrorData}
+     */
+    this.requestData = {
+      jsonData: request.options.data,
```

Comment:
Why not just call it `data`?

Reply:
Because the entire thing is considered data, because if I repeat data one more time I will cry, and mostly to explicitly say that this is JSON data, not form data or such... If Crawl also wants it changed, I will alter accordingly 👍

Reply:
Thenn make the thing `error.data.body`, as opposed to duplicating data twice (`error.requestData.jsonData`)?